### PR TITLE
Slowly Separating Out Client Functions

### PR DESCRIPTION
**Pull request by caleb-mabry**
Moving the logic around for client functions to make them easier to handle and eventually importing functions from other files rather than having one massive `client.js` file. 

This PR also showcases the new/proposed message handler to make adding functions easier in the future and moving message handling into a separate file.
```js
    const messageHandler = {
      MS: client.handleMS,
    };

    if (Object.keys(messageHandler).includes(header)) {
      messageHandler[header](args);
    } else if (!client.emit(header, args)) {
      console.warn(`Invalid packet header ${header}`);
    }
  }
```

### CODE REVIEW

File: webAO/client.js [hunk 1]
```diff
@@ -90,6 +90,46 @@ if (window.requestIdleCallback) {
 
 let lastICMessageTime = new Date(0);
 
+const onMessage = (e) => {
```
Comment (oldmud0) [inline webAO/client.js:93]: these functions belong in the Client class. use `bind()` function to preserve the `this` context

**Review by oldmud0 — CHANGES_REQUESTED** [review]
hey wait caleb no what are you doing you're going the wrong way????

**Comment by caleb-mabry** [comment]
The `this` context is nice, yeah. But I wanted to remove the class entirely from existence and transform it into a functional existence. I suppose this PR is pretty premature and I have a lot more to work on besides moving these small functions.

**Comment by caleb-mabry** [comment]
You technically don't need the this context since everything can be grabbed from the global `client` that's created

**Comment by caleb-mabry** [comment]
Closing this so we can start smaller

**Pull request closed without merging**